contracts/ITokenListManager.sol

```diff
@@ -16,14 +25,16 @@ interface ITokenListManager {
     function removeBlacklistedTokens(address[] calldata tokens) external;
 
     // Get tokens in whitelist, paging supported
-    function getWhitelistedTokens(uint256 offset, uint256 limit) external view returns (address[] memory);
+    function getWhitelistedTokens(uint256 offset, uint256 limit) 
+    external view returns (address[] memory whitelistedERC20Tokens, address[] memory whitelistedERC721Tokens, address[] memory whitelistedERC1155Tokens);
```

Comment:
Do you want to return all tokens in a pagination view?

Basically, returns tokens of specified token type is enough. That is, add a `tokenType` input parameter is ok.

Reply:
Solved